.github/pull_request_template.md

```diff
@@ -22,18 +22,20 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] Test A
 - [ ] Test B
 
+<!---
 **Test Configuration**
 * Hardware:
 * Flutter SDK Version:
 * Flutter Doctor Results:
-
+-->
 ## Is This Backward Compatible?
 
 Are you changes backwards compatible?
```

Comment:
@Mastersam07 plz fix the typo here... it should be `Are your changes backward compatible?`

Reply:
Oh, thanks